File: April-18/21-04-18/c++/01-My-girlfriend-and-her-love-for-cats.cpp
#include<bits/stdc++.h>

using namespace std;
typedef unsigned long long int ll;

int main()
{
    int n;
    cin >> n;
    ll cat[n], cal[n];
    for(int i = 0; i < n; i++)
        cin >> cat[i];
    for(int i = 0; i < n; i++)
        cin >> cal[i];
    
    sort(cat, cat + n);
    sort(cal, cal + n);
    
    ll ans = 0;
    
    for(int i = n - 1; i >= 0; i--)
    {
        ans += (cat[i]*cal[i]);
    }
    cout << ans << endl;
}

File: April-18/21-04-18/python/01-My-girlfriend-and-her-love-for-cats.py
n = int(input())
cat = [int(x) for x in input().split()]
cal = [int(x) for x in input().split()]

cat.sort()
cal.sort()

ans = 0

for i in range(n-1,-1,-1):
    ans += (cat[i]*cal[i])
    
print(ans)
